### ISSUE TITLE
Publish theme to KiCAD addon manager

### ISSUE THREAD

**Issue by Chrismettal** [dracula/kicad#4]
As of KiCAD 6 there is an addon manager included, greatly simplifying the installation process for addons and themes for end users. Maybe it would help to add the theme there.

https://dev-docs.kicad.org/en/addons/

**Comment by miameows** [comment]
@Chrismettal Because this repo seems to be on hiatus, I've made a fork of it (it's the same color pallet) and published it on the PCM. It's under the name "Vampire", feel free to give it a go!

P.S. If the original creators of this theme want to publish it to the PCM, I'd be happy to take my fork down so it could be replaced by an official version! Just use the email on my GitHub profile.

**Comment by Chrismettal** [comment]
We could also talk about transfer of maintainership if this repo is unmaintained. Keeping the official repo alive would be much more beneficial than forking and republishing I would say.

**Comment by miameows** [comment]
I agree, but the creator of the Dracula project would need to get involved for that to happen. I'll see if I can reach out to them.

**Comment by miameows** [comment]
I just made an issue on the main Dracula theme repo, we'll see what happens :) Thanks for the suggestion!

dracula/dracula-theme#882

**Comment by bagnaram** [comment]
I am happy to merge the changes since I believe the repo is still in the `kicad dracula` project.

**Comment by bagnaram** [comment]
Does it make sense to publish a module here, on the KiCAD GitLab or (both)?

**Comment by miameows** [comment]
When I made my fork of this project ([2231puppy/kicad-vampire](https://github.com/2231puppy/kicad-vampire)), I had to do some directory restructuring to make it work with the PCM. There _might_ be a way to get it to work without moving anything around, but I'm not sure. I currently host the fork of this theme on GitHub, but KiCad requires that you submit an MR on GitLab to get the plugin in the PCM. Would it be easier if I submitted a PR, or do you think you've got it?

Let me know if there's anything I can to to help! Thanks so much :)

**Comment by bagnaram** [comment]
Maybe what we do is include your release as the current version and still providing the themes for prior versions of the product. Some of the enterprise users like myself are stuck on the older version of KiCad without addon capability but I love the idea of jumping to the new package format for new versions.

We can start by adding a README entry for the new package available on GitLAB. I say go ahead and merge it to KiCAD GitLAB and here we can point to the instructions. The source of the color definitions can stay here on the official `dracula` repository.
